core/modules/color/color.module

```diff
@@ -369,10 +379,29 @@ function color_get_info($theme) {
 
   if (!isset($color_info[$theme])) {
     $info = array(); // The variable to be populated by the theme color.inc file.
```

Comment:
We'll need to update this inline comment, to mention the new functionality + also move it to the line before the `$info` variable.